app/src/main/java/org/simple/clinic/newentry/form/InputField.kt

```diff
@@ -1,3 +1,5 @@
 package org.simple.clinic.newentry.form
 
-abstract class InputField
+abstract class InputField<T> {
+  abstract fun validate(value: T): List<ValidationError>
```

Comment:
Should this be returning `Set<ValidationError>` instead?

Reply:
Yes, you are right. Will make this change in the upcoming PR?

Reply:
Works.